packages/common/solana/types.ts

```diff
@@ -21,4 +21,5 @@ export interface CheckAssociatedTokenAccountsData {
 export interface AtaParams {
   mint: PublicKey;
   owner: PublicKey;
+  programId?: PublicKey;
```

Comment:
can we use that brand new fn `getMintAndProgram` and resolve it implicitly without exposing this additional parameter for APIs: `ata`/`ataBatchExist`/`createAtaBatch` etc?